.ci/magician/cmd/generate_comment.go

```diff
@@ -265,58 +265,68 @@ func execGenerateComment(prNumber int, ghTokenMagicModules, buildId, buildStep,
 			uniqueBreakingChanges[breakingChange] = struct{}{}
 		}
 
-		// If fetching the PR failed, Labels will be empty
-		labels := make([]string, len(pullRequest.Labels))
-		for i, label := range pullRequest.Labels {
-			labels[i] = label.Name
-		}
-		serviceLabels, err := changedSchemaLabels(prNumber, labels, diffProcessorPath, gh, rnr)
+		affectedResources, err := changedSchemaResources(diffProcessorPath, rnr)
 		if err != nil {
-			fmt.Println("computing changed schema labels: ", err)
-			errors[repo.Title] = append(errors[repo.Title], "The diff processor crashed while computing changed schema labels.")
+			fmt.Println("computing changed resource schemas: ", err)
+			errors[repo.Title] = append(errors[repo.Title], "The diff processor crashed while computing changed resource schemas.")
 		}
-		for _, serviceLabel := range serviceLabels {
-			uniqueServiceLabels[serviceLabel] = struct{}{}
+		for _, resource := range affectedResources {
+			uniqueAffectedResources[resource] = struct{}{}
 		}
 	}
 	breakingChangesSlice := maps.Keys(uniqueBreakingChanges)
 	sort.Strings(breakingChangesSlice)
 	data.BreakingChanges = breakingChangesSlice
 
 	// Compute affected resources based on changed files
-	affectedResources := map[string]struct{}{}
+	changedFilesAffectedResources := map[string]struct{}{}
 	for _, repo := range []source.Repo{tpgRepo, tpgbRepo} {
 		if !repo.Cloned {
 			fmt.Println("Skipping changed file service labels; repo failed to clone: ", repo.Name)
 			continue
 		}
 		for _, path := range repo.ChangedFiles {
 			if r := fileToResource(path); r != "" {
-				affectedResources[r] = struct{}{}
+				uniqueAffectedResources[r] = struct{}{}
+				changedFilesAffectedResources[r] = struct{}{}
```

Comment:
Just want to confirm my understanding is correct - we use `changedFilesAffectedResources` only to log the affected resources based on changed files, and  `uniqueAffectedResources` for converting affected resources(including detected from file changes or schema changes) - so we need to added the affected resource to  both sets here

Reply:
yes - I added the second set only for printing debugging information.